sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -82,6 +82,98 @@ With this configuration, the Azure Monitor Distro will use the credentials of th
 
 Note that the `Credential` property is optional. If it is not set, Azure Monitor Distro will use the Instrumentation Key from the Connection String to send data to Azure Monitor.
 
+### Advanced configuration
```

Comment:
I recommend adding a section just before this header to document what is included in the distro. Which sampler, which instrumentations, which exporters, resource detectors etc. Without knowing what is included, there advanced configuration section lacks context.